HomeTask3/EventDispatcher.js

```diff
@@ -0,0 +1,12 @@
+class EventEmitter {
```

Comment:
I am not sure about this event Emitter class
If you intend to use customEvents you can do that via window object. This class seems redundant for me

Reply:
Removed event emitter class.